supplementary_style_guide/glossary_terms_conventions/product_conventions/eap-conventions/eap-terms-and-usage--b.adoc

```diff
@@ -0,0 +1,56 @@
+= B 
+
+[discrete]
+== Batch Subsystem
+
+*Description* Heading format for the batch-jberet subsystem.
+
+*Use it?* Yes
+
+*Incorrect forms*
+
+*See* batch-jberet
+
+[discrete]
+== batch-jberet
+
+*Description* The JBoss EAP subsystem used to configure and manage batch jobs.  Always lower case except when used in a heading.
+
+*Use it?* Yes
+
+*Incorrect forms*
+
+*See* Batch
+
+[discrete]
+== Bean Validation Subsystem
+
+*Description* Heading format for the bean-validation subsystem.
+
+*Use it?* Yes
+
+*Incorrect forms*
+
+*See* bean-validation
+
+[discrete]
+== bean-validation
+
+*Description* The JBoss EAP subsystem use to configure validation of Java bean object data.
```

Comment:
Should "use to" be "used to"?

Reply:
This is now fixed based on updates to the jboss-eap.adoc file in https://github.com/redhat-documentation/supplementary-style-guide/pull/97.